(10-13) 19:38:58 INFO     [aggregator.py:33] Job args Namespace(adam_epsilon=1e-08, arrival_interval=3, async_buffer=10, async_mode=False, backbone='./resnet50.pth', backend='gloo', batch_size=20, bidirectional=True, blacklist_max_len=0.3, blacklist_rounds=-1, block_size=64, cfg_file='./utils/rcnn/cfgs/res101.yml', checkin_period=50, clf_block_size=32, clip_bound=0.9, clip_threshold=3.0, clock_factor=1.0, conf_path='~/dataset/', connection_timeout=60, cuda_device=None, cut_off_util=0.05, data_cache='', data_dir='/u/utd8hj/Documents/FedScale/benchmark/dataset/data/MHEALTH', data_map_file='/u/utd8hj/Documents/FedScale/benchmark/dataset/data/MHEALTH/client_data_mapping/train.csv', data_set='MHEALTH', decay_factor=0.98, decay_round=10, device_avail_file='/u/utd8hj/Documents/FedScale/benchmark/dataset/data/device_info/client_behave_trace', device_conf_file='/u/utd8hj/Documents/FedScale/benchmark/dataset/data/device_info/client_device_capacity', dump_epoch=10000000000.0, embedding_file='glove.840B.300d.txt', engine='pytorch', epsilon=0.9, eval_interval=10, executor_configs='127.0.0.1:[4]', experiment_mode='simulation', exploration_alpha=0.3, exploration_decay=0.98, exploration_factor=0.9, exploration_min=0.3, filter_less=21, filter_more=1000000000000000.0, finetune=False, gamma=0.9, gradient_policy=None, hidden_layers=7, hidden_size=256, input_dim=22, job_name='MHEALTH_demo', labels_path='labels.json', learning_rate=0.05, line_by_line=False, local_steps=5, log_path='/u/utd8hj/Documents/FedScale/benchmark', loss_decay=0.2, malicious_factor=1000000000000000.0, max_concurrency=100, max_staleness=5, memory_capacity=2000, min_learning_rate=5e-05, mlm=False, mlm_probability=0.15, model='lr', model_size=65536, model_zoo='torchcv', n_actions=2, n_states=4, noise_dir=None, noise_factor=0.1, noise_max=0.5, noise_min=0.0, noise_prob=0.4, num_class=13, num_classes=13, num_executors=4, num_loaders=2, num_participants=50, output_dim=0, overcommitment=1.3, overwrite_cache=False, pacer_delta=5, pacer_step=20, proxy_mu=0.1, ps_ip='127.0.0.1', ps_port='29501', rnn_type='lstm', round_penalty=2.0, round_threshold=30, rounds=200, sample_mode='random', sample_rate=16000, sample_seed=233, sample_window=5.0, spec_augment=False, speed_volume_perturb=False, target_delta=0.0001, target_replace_iter=15, task='cv', test_bsz=20, test_manifest='data/test_manifest.csv', test_output_dir='./logs/server', test_ratio=1.0, test_size_file='', this_rank=0, time_stamp='1013_193855', train_manifest='data/train_manifest.csv', train_size_file='', train_uniform=False, upload_step=20, use_cuda=True, vocab_tag_size=500, vocab_token_size=10000, weight_decay=0, window='hamming', window_size=0.02, window_stride=0.01, yogi_beta=0.9, yogi_beta2=0.99, yogi_eta=0.003, yogi_tau=1e-08)
2022-10-13 19:38:58.934220: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F AVX512_VNNI FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-10-13 19:38:59.078220: I tensorflow/core/util/util.cc:169] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2022-10-13 19:38:59.082287: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2022-10-13 19:38:59.082299: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2022-10-13 19:38:59.103799: E tensorflow/stream_executor/cuda/cuda_blas.cc:2981] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2022-10-13 19:39:00.417831: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory
2022-10-13 19:39:00.417955: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory
2022-10-13 19:39:00.417969: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
(10-13) 19:39:02 INFO     [aggregator.py:129] Initiating control plane communication ...
(10-13) 19:39:02 INFO     [aggregator.py:153] %%%%%%%%%% Opening aggregator sever using port [::]:29501 %%%%%%%%%%
(10-13) 19:39:02 INFO     [fllibs.py:119] Initializing the model ...
(10-13) 19:39:03 INFO     [aggregator.py:871] Start monitoring events ...
(10-13) 19:39:09 INFO     [executor.py:61] (EXECUTOR:1) is setting up environ ...
(10-13) 19:39:11 INFO     [executor.py:61] (EXECUTOR:2) is setting up environ ...
(10-13) 19:39:11 INFO     [executor.py:117] Data partitioner starts ...
(10-13) 19:39:11 INFO     [divide_data.py:63] Partitioning data by profile /u/utd8hj/Documents/FedScale/benchmark/dataset/data/MHEALTH/client_data_mapping/train.csv...
(10-13) 19:39:11 INFO     [divide_data.py:75] Trace names are client_Id, chest_acc_x, chest_acc_y, chest_acc_z, ecg_1, ecg_2, left_ankle_acc_x, left_ankle_acc_y, left_ankle_acc_z, left_ankle_gyro_x, left_ankle_gyro_y, left_ankle_gyro_z, left_ankle_mag_x, left_ankle_mag_y, left_ankle_mag_z, right_arm_acc_x, right_arm_acc_y, right_arm_acc_z, right_arm_gyro_x, right_arm_gyro_y, right_arm_gyro_z, right_arm_mag_x, right_arm_mag_y, right_arm_mag_z, label_Id
(10-13) 19:39:12 INFO     [divide_data.py:106] Randomly partitioning data, 24333 samples...
(10-13) 19:39:12 INFO     [executor.py:128] Data partitioner completes ...
(10-13) 19:39:12 INFO     [channel_context.py:21] %%%%%%%%%% Opening grpc connection to 127.0.0.1 %%%%%%%%%%
(10-13) 19:39:12 INFO     [executor.py:414] Start monitoring events ...
(10-13) 19:39:12 INFO     [aggregator.py:263] Received executor 1 information, 1/4
(10-13) 19:39:13 INFO     [executor.py:61] (EXECUTOR:3) is setting up environ ...
(10-13) 19:39:13 INFO     [executor.py:117] Data partitioner starts ...
(10-13) 19:39:13 INFO     [divide_data.py:63] Partitioning data by profile /u/utd8hj/Documents/FedScale/benchmark/dataset/data/MHEALTH/client_data_mapping/train.csv...
(10-13) 19:39:13 INFO     [divide_data.py:75] Trace names are client_Id, chest_acc_x, chest_acc_y, chest_acc_z, ecg_1, ecg_2, left_ankle_acc_x, left_ankle_acc_y, left_ankle_acc_z, left_ankle_gyro_x, left_ankle_gyro_y, left_ankle_gyro_z, left_ankle_mag_x, left_ankle_mag_y, left_ankle_mag_z, right_arm_acc_x, right_arm_acc_y, right_arm_acc_z, right_arm_gyro_x, right_arm_gyro_y, right_arm_gyro_z, right_arm_mag_x, right_arm_mag_y, right_arm_mag_z, label_Id
(10-13) 19:39:14 INFO     [divide_data.py:106] Randomly partitioning data, 24333 samples...
(10-13) 19:39:14 INFO     [executor.py:128] Data partitioner completes ...
(10-13) 19:39:14 INFO     [channel_context.py:21] %%%%%%%%%% Opening grpc connection to 127.0.0.1 %%%%%%%%%%
(10-13) 19:39:14 INFO     [executor.py:414] Start monitoring events ...
(10-13) 19:39:14 INFO     [aggregator.py:263] Received executor 2 information, 2/4
(10-13) 19:39:15 INFO     [executor.py:61] (EXECUTOR:4) is setting up environ ...
(10-13) 19:39:16 INFO     [executor.py:117] Data partitioner starts ...
(10-13) 19:39:16 INFO     [divide_data.py:63] Partitioning data by profile /u/utd8hj/Documents/FedScale/benchmark/dataset/data/MHEALTH/client_data_mapping/train.csv...
(10-13) 19:39:16 INFO     [divide_data.py:75] Trace names are client_Id, chest_acc_x, chest_acc_y, chest_acc_z, ecg_1, ecg_2, left_ankle_acc_x, left_ankle_acc_y, left_ankle_acc_z, left_ankle_gyro_x, left_ankle_gyro_y, left_ankle_gyro_z, left_ankle_mag_x, left_ankle_mag_y, left_ankle_mag_z, right_arm_acc_x, right_arm_acc_y, right_arm_acc_z, right_arm_gyro_x, right_arm_gyro_y, right_arm_gyro_z, right_arm_mag_x, right_arm_mag_y, right_arm_mag_z, label_Id
(10-13) 19:39:16 INFO     [divide_data.py:106] Randomly partitioning data, 24333 samples...
(10-13) 19:39:16 INFO     [executor.py:128] Data partitioner completes ...
(10-13) 19:39:16 INFO     [channel_context.py:21] %%%%%%%%%% Opening grpc connection to 127.0.0.1 %%%%%%%%%%
(10-13) 19:39:16 INFO     [executor.py:414] Start monitoring events ...
(10-13) 19:39:16 INFO     [aggregator.py:263] Received executor 3 information, 3/4
(10-13) 19:39:17 INFO     [executor.py:117] Data partitioner starts ...
(10-13) 19:39:17 INFO     [divide_data.py:63] Partitioning data by profile /u/utd8hj/Documents/FedScale/benchmark/dataset/data/MHEALTH/client_data_mapping/train.csv...
(10-13) 19:39:17 INFO     [divide_data.py:75] Trace names are client_Id, chest_acc_x, chest_acc_y, chest_acc_z, ecg_1, ecg_2, left_ankle_acc_x, left_ankle_acc_y, left_ankle_acc_z, left_ankle_gyro_x, left_ankle_gyro_y, left_ankle_gyro_z, left_ankle_mag_x, left_ankle_mag_y, left_ankle_mag_z, right_arm_acc_x, right_arm_acc_y, right_arm_acc_z, right_arm_gyro_x, right_arm_gyro_y, right_arm_gyro_z, right_arm_mag_x, right_arm_mag_y, right_arm_mag_z, label_Id
(10-13) 19:39:17 INFO     [divide_data.py:106] Randomly partitioning data, 24333 samples...
(10-13) 19:39:17 INFO     [executor.py:128] Data partitioner completes ...
(10-13) 19:39:17 INFO     [channel_context.py:21] %%%%%%%%%% Opening grpc connection to 127.0.0.1 %%%%%%%%%%
(10-13) 19:39:17 INFO     [executor.py:414] Start monitoring events ...
Traceback (most recent call last):
  File "/u/utd8hj/Documents/FedScale/fedscale/core/execution/executor.py", line 455, in <module>
    executor.run()
  File "/u/utd8hj/Documents/FedScale/fedscale/core/execution/executor.py", line 143, in run
    self.event_monitor()
  File "/u/utd8hj/Documents/FedScale/fedscale/core/execution/executor.py", line 426, in event_monitor
    client_id, train_res = self.Train(train_config)
  File "/u/utd8hj/Documents/FedScale/fedscale/core/execution/executor.py", line 205, in Train
    clientId=client_id, conf=client_conf, model=model)
  File "/u/utd8hj/Documents/FedScale/fedscale/core/execution/executor.py", line 319, in training_handler
    client_model = self.load_global_model() if model is None else model
  File "/u/utd8hj/Documents/FedScale/fedscale/core/execution/executor.py", line 274, in load_global_model
    with open(self.temp_model_path, 'rb') as model_in:
FileNotFoundError: [Errno 2] No such file or directory: '/u/utd8hj/Documents/FedScale/benchmark/logs/MHEALTH_demo/1013_193855/executor/model_4.pth.tar'
Permission denied, please try again.
Permission denied, please try again.
utd8hj@127.0.0.1: Permission denied (publickey,password).
